packages/name-controller/src/NameController.test.ts

```diff
@@ -109,6 +118,7 @@ describe('NameController', () => {
             [CHAIN_ID_MOCK]: {
               name: null,
               sourceId: null,
+              origin: NameOrigin.API,
```

Comment:
Minor, but it is conceptually clearer to leave the `origin` as `null` in the state in these cases where there is no `name`?

Reply:
Yes, good catch.